app.py

```diff
@@ -3,24 +3,55 @@
 from flask import Flask
 from flask_restplus import Api, Resource
 from flask_sqlalchemy import SQLAlchemy
+from datetime import datetime
 
 app = Flask(__name__)
-app.config["SQALCHEMY_DATABASE_URL"] = os.getenv("APP_SQL_URL")
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("APP_SQL_URL")
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 db = SQLAlchemy(app)
 api = Api(app, title="Review Requester")
 
-
 class Reviewer(db.Model):
-    pass
+    __tablename__ = 'reviewer'
+
+    id = db.Column(db.Integer, primary_key=True)
+    first_name = db.Column(db.String(50), unique=True, nullable=False)
+    last_name = db.Column(db.String(50), unique=True, nullable=False)
+    email_address = db.Column(db.String(50), unique=True, nullable=False)
 
+    __table_args__ = (db.UniqueConstraint('last_name', 'first_name', name='name_uix'), )
+
+    def __repr__(self):
+        return '<User %r>' % self.id
 
 class ReviewLanguage(db.Model):
-    pass
+    __tablename__ = 'review_language'
 
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(50), unique=True, nullable=False)
+
+    def __repr__(self):
+        return '<User %r>' % self.id
```

Comment:
Looks like an accidental copypasta - might be useful to have the name here instead as well.
```suggestion
        return '<Language %s>' % self.name
```

Reply:
fixed